lib/tag-finder.coffee

```diff
@@ -8,14 +8,14 @@ SelfClosingTags = require './self-closing-tags'
 module.exports =
 class TagFinder
   constructor: (@editor) ->
-    @tagPattern = /(<(\/?))([^\s>]+)([\s>]|$)/
-    @wordRegex = /[^>\r\n]*/
+    @tagPattern = /(<(\/?))([^\s>]+)([^/>]*)(\/?)(>)/
```

Comment:
I don't know how to fix this, but `/(<(\/?))([^\s>]+)([^/>]*)(\/?)(>)/` won't match `<foo bar="/">`. This is because `[^/>]` dose not mean "not the sequence `/>`" but match all characters but not `/` or `>`. Same goes for the regex in `patternForTagName`.